psamm_import/importer.py

```diff
@@ -124,6 +124,26 @@ def decimal_representer(dumper, data):
     return dumper.represent_scalar('tag:yaml.org,2002:float', value)
 
 
+def detect_extracellular(model):
+    """Detect the identifier for equations with extracellular compartments,
+        as it's not always the same.
+    """
```

Comment:
Summary docstring should be a single line. I would probably just shorten it to `Detect the identifier for equations with extracellular compartments.`